perf/pgbench.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2023 IBM
+# Author: Gautam Menghani <gautam@linux.ibm.com>
+
+import os
+
+from avocado import Test
+from avocado.utils import process, cpu
+from avocado.utils.service import ServiceManager
+from avocado.utils.software_manager.manager import SoftwareManager
+
+
+class Pgbench(Test):
+
+    """
+    This module will run the pgbench benchmark. Pgbench is a benchmark for
+    measuring the throughput achieved by executing database transactions on
+    PostgresSQL database.
+    """
+    sm = SoftwareManager()
+
+    def run_cmd(self, cmd, ignore_failure):
+        command = process.SubProcess(cmd)
+        command.run()
+        if command.result.exit_status:
```

Comment:
this if else ladder can be simplified

if ignore_failure:
   return [command.result.exit_status]
elif command.result.exit_status:
   self.cancel()
else:
   return [exit_status..]